src/styles/components/_introduction.scss

```diff
@@ -1,3 +1,65 @@
 .introduction {
+  display: flex;
+  align-items: center;
+  color: white;
   background-color: $color-black;
+
+  &__container {
+    @include container;
+  }
+
+  &__img {
+    display: none;
+
+    @media #{$screen-sm} {
+      display: block;
+    }
+  }
+
+  &__header {
+    margin: 30px 0;
+    font-family: 'Rubik Mono One', sans-serif;
```

Comment:
Maybe move this font to a variable or add a 'TODO' comment, to fix it after I merge my branch, because I already did it there.